readme.adoc

```diff
@@ -50,6 +50,7 @@ as you move through the workshop.
 
 | link:statefulsets[StatefulSets with EBS]
 | link:calico[Network policies with Calico]
+| link:weavenet[Network policies with Weave Net]
```

Comment:
Create a top-level networking directory, refactor `calico` to that directory. Also move `weavenet` there as well.